CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 **v5.6.0(unreleased):**
 
 * final_grades.csv: notebook name is the file name for each row instead of the full path to the notebook and the rows are sorted by notebook name
+* added wget to grade/Dockerfile -- this allows otter-grader version 4 generated autograder.zips to work with otter-grader version 5
```

Comment:
generally I prefer to leave implementation details out of the changelog since it has a wide audience; can we reword this to something like "Added backwards compatibility to Otter Grade for autograder configuration zip files generated in previous major versions of Otter-grader"

Reply:
Cool. Sent it through. Thanks!